client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -22,20 +24,37 @@ const DesignSelector: FunctionComponent< Props > = () => {
 	const { selectedDesign, siteVertical } = useSelect( select =>
 		select( ONBOARD_STORE ).getState()
 	);
-	const { setSelectedDesign } = useDispatch( ONBOARD_STORE );
+	const { setSelectedDesign, resetOnboardStore } = useDispatch( ONBOARD_STORE );
+	const makePath = usePath();
+
+	const handleStartOverButtonClick = () => {
+		resetOnboardStore();
+	};
 
 	return (
 		<div
 			className="design-selector gutenboarding-color-coded"
 			data-vertical={ siteVertical?.label }
 		>
 			<div className="design-selector__header">
-				<h1 className="design-selector__title">{ NO__( 'Choose a starting design' ) }</h1>
-				<h2 className="design-selector__subtitle">
-					{ NO__(
-						'Get started with one of our top website layouts. You can always change it later'
-					) }
-				</h2>
+				<div className="design-selector__header-section design-selector__header-section--title">
```

Comment:
`design-selector__header-section--title` I think we don't have this format in CSS selectors. It's usually with single dash `-title`.

Since `design-selector__header-section` isn't used in CSS, I think we can simplify this somehow.

Reply:
> design-selector__header-section--title I think we don't have this format in CSS selectors. It's usually with single dash -title. I might be wrong too, there's prolly a document somewhere in Calypso explaining these. 😁

Ah okay! Initially I thought we're using BEM but our styleguide is actually here:
https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md

> Since design-selector__header-section isn't used in CSS, I think we can simplify this somehow.

Yeah I'll think of how to simplify this further.

Reply:
Done.